packages/theming/win32-theme/src/createPartialOfficeTheme.ts

```diff
@@ -33,6 +33,6 @@ export function createPartialOfficeTheme(module: OfficeThemingModule, themeName?
       // Office Branding Colors
       colors: getRamps(module),
     },
-    ...(themeName && { name: themeName }),
+    ...(!!themeName ? { name: themeName } : undefined),
```

Comment:
Doesn't seem right that macOS is using the `win32-theme` file

Reply:
I agree, but it looks tied in with simply using the Fluent Button:

```
createPartialOfficeTheme (index.js:formatted:523)
translateOfficeTheme (index.js:formatted:607)
getPlatformDefaults (index.js:formatted:621)
createPlatformThemeRegistry (index.js:formatted:828)
getThemeRegistry (index.js:formatted:846)
getTheme (index.js:formatted:891)
useStylingCore (index.js:formatted:1188)
(anonymous) (index.js:formatted:1213)
usePrepareProps (Button.tsx?4e81:49)
useUpdateRenderData (Composable.slots.ts?63de:87)
useCompoundPrepare (Composable.slots.ts?63de:120)
Button (Composable.ts?a5ad:99)
renderWithHooks (ReactNativeRenderer-dev.js?2a1c:10611)
```

At the moment, I'm just trying to unblock newer versions of Fluent on macOS -- from what I can tell, it's always been this way (even before the commit I indicated in my notes earlier). Folks can refactor\rework things as they see fit separately to remove this dependency.

Reply:
Yeah, I think he only "theme" provided right now is the win32 / office theme, and we just haven't built out platform specific / other themes yet. Definitely should be on our list of things to do, would help with making the test apps look/feel more native.

Reply:
Should we not go ahead with the current change that restores previous behavior (prior to commit 59d0857d4244fb6b35fb2c9ca7d01cf2a073f461)? If not, what should I be doing?

As I mentioned, macOS is currently blocked on adopting newer versions of Fluent. I need the same SDX on this newer version of Fluent because I need the capabilities exposed therein on Win32. Fixing the block (which was introduced two weeks ago) feels more pressing to me than fixing a problem that has probably existed since a while (months, I'll guess).

Reply:
It seems fine to me, but I'm less familiar with this part of the code and wanted someone more familiar to review. The reviewers you initially added should be good.

Reply:
The change looks good and the win32-theme dependency is something I/we are in the process of removing. I think one of the PRs I have out will actually remove this dependency. It was from theming-react-native containing the native module and being referenced from foundation-compose I believe. I think I changed the framework dependencies to only be theme-types and more generic packages, then made the win32/office theming parts optional in the tester.